tools/nni_cmd/nnictl_utils.py

```diff
@@ -151,7 +151,7 @@ def parse_ids(args):
             exit(1)
         else:
             result_list = running_experiment_list
```

Comment:
do you need to consider change line 554 and line 434? because "args_all" look likes a boolean variable in these lines.

Reply:
In the `nnictl.py`, `all` is treated as one value of args.id under the `stop_experiment` subparser but is a boolean value `--all` under other subparser so they are different. According to the current document, user need to type something like `nnictl list experiment --all` to list all experiments. But they only need to type `nnictl stop all` to stop all running experiment. 
Originally I'd like to change the command to `nnictl stop --all` so they might be a little more consistent but I think that it has become customary to type `nnictl stop all` so I discard that thought.